### ISSUE TITLE
Missing Helper Cake\View\Exception\MissingHelperException

### ISSUE THREAD

**Issue by AquilaeNox** [ctlabvn/Recaptcha#19]
I've done everything as requested but I'm getting Missing Helper
Cake\View\Exception\MissingHelperException

As far as I understand the display() method is generating this error simply because the helper RecaptchaHelper.php probably is not being loaded, not sure why but when I register the helper under AppView.php as:

```
public function initialize()
{
        $this->helpers = [
            'Recaptcha' => [
                'className' => 'Recaptcha',
            ],
        ];
}
```

I'm getting: Helper class RecaptchaHelper could not be found.

**Comment by AquilaeNox** [comment]
I can only address this behaviour to the RecaptchaComponent initialize() method

**Comment by ADmad** [comment]
`'className' => 'Recaptcha',` should be `'className' => 'Recaptcha.Recaptcha',`.
You need to use the plugin prefix.

**Comment by AquilaeNox** [comment]
Yes that solved the helper loading issue but thats it.
I am ware that in dev production the captcha will not work (Invalid domain for site key) but that's not the problem. The display() method will generate a notice and debug($this->getConfig()) only contains:

`'className' => 'Recaptcha.Recaptcha'`

And: `<?= $this->Recaptcha->display()?>`
Displays: `Notice (8): Undefined index: enable [ROOT/vendor/crabstudio/recaptcha/src/View/Helper/RecaptchaHelper.php, line 29]`
 
I've tried to load the component under AppController -> initialize or even at the controller initialize method (where I will need the captcha). I also tried to pass the config through method parameter or even set it up under $_defaultConfig (RecaptchaComponent.php) but none has worked for me. So, I've manually added the script at my layout in order to test and it worked without and issue.

I'm thinking I'm doing something quite bad but I don't have any clue on what possibly can be wrong
